app/src/main/java/com/sebastianvm/mango/data/UserStore.kt

```diff
@@ -0,0 +1,36 @@
+package com.sebastianvm.mango.data
+
+import com.sebastianvm.mango.database.MangoDatabase
+import com.sebastianvm.mango.database.models.UserEnt
+import com.sebastianvm.mango.model.User
+import com.sebastianvm.mango.util.coroutines.IODispatcher
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.components.ViewModelComponent
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.withContext
+
+class UserStore (
```

Comment:
To be able to use fakes, we would want to create interfaces on the repository layer, too. I'm thinking that for each repository, we create a package with:
* `MyRepository.kt`: interface definition
* `MyRepositoryImpl.kt`: implementation
* `MyRepositoryModule.kt`:  Hilt module definition

Reply:
yup you're totally right